.circleci/config.yml

```diff
@@ -94,7 +94,7 @@ jobs:
                     pip install -r tests/requirements.txt --quiet
                     npm install --ignore-scripts
                     npm run build:all
-                    pytest tests/test_usage.py --driver Chrome
+                    pytest tests/test_usage.py --webdriver Chrome
```

Comment:
is the `webdriver` setting not picked up from the `{{cookiecutter.project_shortname}}/pytest.ini` file?

Reply:
It should :+1: